src/main/java/examples/VertxMqttServerExamples.java

```diff
@@ -267,11 +293,44 @@ public void example10(Vertx vertx) {
 
   /**
    * Example for scaling (sharing MQTT servers)
+   *
    * @param vertx
    */
   public void example11(Vertx vertx) {
 
     DeploymentOptions options = new DeploymentOptions().setInstances(10);
     vertx.deployVerticle("com.mycompany.MyVerticle", options);
   }
+
+  /**
+   * Example for server use proxy protocol<br/>
+   * <p>
+   * You need to set the port of <code>haproxy</code> or <code>nginx</code> to proxy port 1883 and
+   * enable the <code>proxy protocol</code> function
+   *
+   * @param vertx
+   */
+  public void example12(Vertx vertx) {
+    MqttServer mqttServer = MqttServer
+      .create(vertx, new MqttServerOptions()
+        // set true to use proxy protocol
+        .setUseProxyProtocol(true));
+    mqttServer.endpointHandler(endpoint -> {
+      // remote address is origin real address， not proxy's address
+      System.out.println(endpoint.remoteAddress());
+      endpoint.accept(false);
+
```

Comment:
empty line

Reply:
well, This code turns out to be like this. Did you mean you want to remove this empty line now?